apps/Routes/routes.py

```diff
@@ -2,7 +2,7 @@
 from apps.Timelapse.routes import routes as tRoutes
 from apps.ImageService.routes import routes as isRoutes
 from apps.PiCam.routes import routes as pcRoutes
-from apps.TelemData.routes import routes as telemRoutes
+from apps.Pixhawk.routes import routes as telemRoutes
```

Comment:
Non-blocking, but, @dragonprevost Do we even use this Routes file anymore since turfing `tornado`?

Reply:
Yeah this should get wiped once merged with master. I will run a simulated merge to make sure we don't get too many conflicts

Reply:
I just looked into this and the routes application still exists in master but lets just delete it after we merge this PR...